src/lib.rs

```diff
@@ -85,7 +85,7 @@ pub fn execute_piranha(piranha_arguments: &PiranhaArguments) -> Vec<PiranhaOutpu
   let source_code_units = piranha.get_updated_files();
 
   for scu in source_code_units.iter() {
-    scu.persist(piranha_arguments);
```

Comment:
we were unnecessarily passing `piranha_arguments` (`SourceCodeUnit` already has `piranha_arguments`). So I updated this signature. 

This is the only signature I changed.